src/etl/impl/LedgerPublisher.h

```diff
@@ -69,10 +72,12 @@ class LedgerPublisher
     LedgerPublisher(
         boost::asio::io_context& ioc,
         std::shared_ptr<BackendInterface> backend,
-        std::shared_ptr<feed::SubscriptionManager> subscriptions,
+        std::reference_wrapper<CacheType> cache,
```

Comment:
minor: We usually pass argument by reference and then use `std::ref` or `std::cref` (see SystemState below). Consider changing for consistency 👍